docs/04-api-reference/front-commerce-core/ServerEvent.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "ServerEvent"
+sidebar_position: 8
+description:
+  "The ServerEvent class define ServerEvent format used in Front-Commerce to be
+  published to the server event worker."
+---
+
+<p>{frontMatter.description}</p>
+
+## Usage:
+
+```ts
+import { ServerEvent } from "@front-commerce/core/events";
+
+// We define a static event Name for our event
+const MyEventName = "MyEvent";
```

Comment:
**question:** Why declaring it in an external `const` rather than inlining it directly where it's used, since it is (and should be) only used one time?

**suggestion:** Remove this declaration and hardcode it directly in the constructor.